school/src/main/java/com/hogwarts/school/controller/FacultyController.java

```diff
@@ -41,17 +41,21 @@ public ResponseEntity<Faculty> updateFaculty(@RequestBody Faculty faculty) {
         }
         return ResponseEntity.ok(fcl);
     }
-    @DeleteMapping("/removeFaculty/{facultyId}")
-    public ResponseEntity<Faculty> removeStudent(@PathVariable Long facultyId) {
-        Faculty fcl = facultyService.removeFaculty(facultyId);
-        if (fcl == null) {
-            return ResponseEntity.notFound().build();
-        }
-        return ResponseEntity.ok(fcl);
+
+    @DeleteMapping("removeFaculty/{facultyId}")
```

Comment:
```suggestion
    @DeleteMapping("/{facultyId}")
```
не нужно ничего писать в пути - это ошибка REST